### PR TITLE
fix message from sql kernel

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

**Review by LadyNaggaga — COMMENTED** [review]
This explanation is a lot clearer

File: src/Microsoft.DotNet.Interactive/SqlKernel.cs [hunk 1]
```diff
@@ -48,21 +48,21 @@ public Task HandleAsync(SubmitCode command, KernelInvocationContext context)
             if (mssqlKernelNames.Count == 0)
             {
                 context.Display(@"
-A SQL kernel is not currently defined.
+A SQL connection has not been established.
```
Comment (chlafreniere) [inline src/Microsoft.DotNet.Interactive/SqlKernel.cs:51]: What's the loc story out of curiosity?

Comment (colombod) [reply src/Microsoft.DotNet.Interactive/SqlKernel.cs]: We do not have one at the moment

**Comment by colombod** [comment]
> This explanation is a lot clearer

All thanks to @jonsequitur

**Review by chlafreniere — APPROVED** [review]

**Review by jonsequitur — APPROVED** [review]

**Pull request merged**